### ISSUE TITLE
Documentation for Init event shows incorrect argument (should be event,slick)

### ISSUE THREAD

**Issue by Samsonator** [kenwheeler/slick#3521]
I noticed while trying to use the "init" trigger that the documentation on http://kenwheeler.github.io/slick/ says that the only argument passed to this function is slick but in reality it is (event, slick). Hope this helps somebody! It could also say that the code should be placed before the .slick initialisation as I've seen a few support questions about that!

```$('.mySlider').on('init', function(event, slick) {```

**Comment by domints** [comment]
I agree it could say that event code should be placed before slick initialisation.

However, I cannot agree on the arguments issue - as you can see it is exactly the same with all events - listing shows arguments without `event`, yet in the example at the top of events section all of them contain `event` argument.

**Comment by Samsonator** [comment]
@domints I can see that but below that events like "Destroy" say explicitly that it passes the event object but if you're saying that all events pass event then it makes it a bit more confusing. Happy to go with your thoughts! It was the inconsistency that initially confused me and I thought it might help other people out!

**Comment by TwoPaperdolls** [comment]
Thanks for pointing this out @Samsonator!

I was going nuts trying to figure out why my console kept saying that slick was undefined when the docs clearly state that slick is the only argument.  Popped event in there and I'm right as rain.
